learningresources/tests/test_api.py

```diff
@@ -61,6 +64,10 @@ def test_create(self):
             create_course(**self.kwargs)
         self.assertTrue(course_count() == after)
 
+        # NOT NULL constraint fails on org
+        with self.assertRaises(IntegrityError):
```

Comment:
Checking for "org" in both cases would be better. A `NULL` could appear for any number of reasons.